package.json

```diff
@@ -16,7 +16,7 @@
         "assert-plus": "^1.0.0",
         "backoff": "2.3.0",
         "bignum": "0.13.0",
-        "boray": "git+https://github.com/joyent/node-buckets-mdapi.git#v0.1.0",
+        "boray": "git+https://github.com/joyent/node-buckets-mdapi.git#MANTA-4725",
```

Comment:
this will have to be changed when the node-boray changes are merged to go back to a tagged release